indigo/docs/platform/importers.md

```diff
@@ -21,7 +21,7 @@ Graphics can be exported as any web compatible file format. One thing to be awar
 
 Animations can also be exported as a spritesheet with accompanying JSON data, here is an example of the export window.
 
-**PLEASE NOTE:** The export is done as a "hash", Indigo does not support the other "array" option.
+**PLEASE NOTE:** The export is done as an "array", Indigo does not support the other "hash" option.
```

Comment:
Well spotted - I think you're right!

Here is an export I use in one of the demos:
https://github.com/PurpleKingdomGames/indigo/blob/main/demos/pirate/assets/Ship%20Helm.json

And here is a json-array export example:
https://gist.github.com/dacap/a32adb9248320326733a
from their [docs](https://www.aseprite.org/docs/cli?l=You%20can%20convert%20or%20export%20your%20sprites%20to%20other%20formats%20(or%20textures%2Bjson%20data)%20from%20the%20command%20line.%20%20See%20Platform-specific%20Details%20section%20to%20know%20how%20to%20use%20the%20command%20line.%0A%0AOptions%0AUse%20Cases%0APlatform-specific%20Details%0AAutomating%20the%20process%0A%0A%0AOpt...&t=json#format).